tasmota/support_device_groups.ino

```diff
@@ -368,13 +385,18 @@ void SendReceiveDeviceGroupMessage(struct device_group * device_group, struct de
       log_remaining--;
       switch (item) {
         case DGR_ITEM_POWER:
-          if (device_group->local) {
+          if (Settings.flag4.remote_device_mode) {  // SetOption88 - Enable relays in separate device groups
+            bool on = (value & 1);
+            if (on != (power & 1)) ExecuteCommandPower(device_group_index + 1, (on ? POWER_ON : POWER_OFF), SRC_REMOTE);
+          }
+          else if (device_group->local) {
             uint8_t mask_devices = value >> 24;
             if (mask_devices > devices_present) mask_devices = devices_present;
-            for (uint32_t i = 0; i < devices_present; i++) {
+            for (uint32_t i = 0; i < mask_devices; i++) {
               uint32_t mask = 1 << i;
               bool on = (value & mask);
               if (on != (power & mask)) ExecuteCommandPower(i + 1, (on ? POWER_ON : POWER_OFF), SRC_REMOTE);
+              if (Settings.flag4.remote_device_mode) break; // SetOption88 - Enable relays in separate device groups
```

Comment:
We should declare and use self-explanatory constants for the options ```Settings.flag4.remote_device_mode)```so that the code is readable without explanatory comments.